docs/docs.less

```diff
@@ -109,12 +104,6 @@ body.hxVertical {
         margin-bottom: 0;
       }
     }
-
-    // PROTOTYPE: vertical spacing
-    // nested to prevent matching the global nav
-    header {
-      margin-bottom: 1rem;
```

Comment:
removed prototype style. it was having adverse affects on numerous pages and styles